packages/@ourworldindata/grapher/src/modal/DownloadModal.tsx

```diff
@@ -201,18 +218,18 @@ export class DownloadTab extends React.Component<DownloadTabProps> {
     }
 
     private renderReady(): JSX.Element {
-        const { targetWidth, targetHeight, svgPreviewUrl, bounds } = this
-
+        const { manager, svgPreviewUrl, tabBounds, targetWidth, targetHeight } =
+            this
         const pngPreviewUrl = this.pngPreviewUrl || this.fallbackPngUrl
 
         let previewWidth: number
         let previewHeight: number
         const boundScalar = 0.17
-        if (bounds.width / bounds.height > targetWidth / targetHeight) {
-            previewHeight = bounds.height * boundScalar
+        if (tabBounds.width / tabBounds.height > targetWidth / targetHeight) {
+            previewHeight = Math.min(72, tabBounds.height * boundScalar)
```

Comment:
Note for the reviewers: We don't want the preview image to be too large since the max-width of the download modal is fixed.